lib/compress/zstd_compress.c

```diff
@@ -2394,6 +2324,81 @@ static void ZSTD_overflowCorrectIfNeeded(ZSTD_matchState_t* ms, ZSTD_CCtx_params
     }
 }
 
+static void ZSTD_copyBlockSequences(const seqStore_t* seqStore, seqDef* seqs,
```

Comment:
minor: prototype style : parameter order, 
generally prefer the mutable target state first, then the arguments (like immutable input).
like `memcpy(out, in, size)`, or `ZSTD_compress(outPtr, outSize, inPtr, inSize, lvl)` .

Reply:
Also: is there any need for `seqDef* seqs` to be mutable ?
It seems to be read only,
in which case prefer `const seqDef* seqs`.

Reply:
Why do we need to pass in `seqs`? Can we just get them from the `seqStore`?